clients/python/test/e2e/conftest.py

```diff
@@ -93,13 +33,13 @@ def configuration() -> Iterable[osparc.Configuration]:
 
 @pytest.fixture
 def api_client(configuration: osparc.Configuration) -> Iterable[osparc.ApiClient]:
-    with osparc.ApiClient(configuration=configuration) as api_client:
-        yield api_client
+    with osparc.ApiClient(configuration=configuration) as client:
+        yield client
```

Comment:
Detail: I actually prefer `api_client` to `client` here because the name of the keyword argument in the automatically generated client is always `api_client`.